README.md

```diff
@@ -16,7 +16,7 @@ Start accepting online payments in just a few minutes with our Android SDK. It's
 ## Requirements
 - Android minimum SDK 21
 
-> Compatibility verified with `targetSdk` versions 21 to 33
+> Compatibility verified with `targetSdk` versions 21 to 34
```

Comment:
It would be nice to update the SDK's `targetSdk` version as 34 as well since we are updating readme. Currently, it is `31`.

Reply:
Does it block merchants' integration for Frames, if we are not upgrading `targetSdk` version to 34 from the SDK? If yes, we can do the release even for one line change as well. My point from the above comment is we should not document anything publicly in readme that we don't support/add. 

However, we can do x number of releases including any changes, I will leave it up to @patrick-hoban-cko